bats.yaml

```diff
@@ -28,9 +26,27 @@ pipeline:
       expected-commit: 5da66876b8b619235aee1eb3e54954eaca88059b
 
   - runs: |
-      ./install.sh  "${{targets.contextdir}}/usr"
+      ./install.sh "${{targets.contextdir}}/usr"
+      mkdir -p "${{targets.contextdir}}/tmp/"
+      cp -r ./docker "${{targets.contextdir}}/tmp/docker"
```

Comment:
https://github.com/bats-core/bats-core/blob/89a7fae3b651d5c2f42a3874ec32e821014814eb/docker/install_libs.sh#L32 I saw that it downloads the files from their own repositories like https://github.com/bats-core/bats-support, so creating a separate packages for them like support, file, assert and detik would be great cuz in that case it'd be easier to keep these things in sync when there is a new version come up, but I'm not certain though.

Reply:
you mean like subpackages or completely new packages?

Reply:
I mean completely new packages, yeah but we can do that later on for sure.

Reply:
I can make an internal dev issue for it to track and get it in!